tests/integration/articleLikes.test.js

```diff
@@ -0,0 +1,224 @@
+import chai, { expect } from "chai";
+import chaiHttp from "chai-http";
+import app from "../../index";
+import { article, users } from "../helpers/testData";
+
+chai.use(chaiHttp);
+const { dummyUser } = users;
+describe("ArticleLike Controller", () => {
+  let slug;
+  let token;
+  before(async () => {
+    await chai
+      .request(app)
+      .post("/api/v1/users")
+      .send({ ...dummyUser });
+    const results = await chai
+      .request(app)
+      .post("/api/v1/users/login")
+      .send({
+        email: dummyUser.email,
+        password: dummyUser.password
+      });
+    ({ token } = results.body);
+    const articleResult = await chai
+      .request(app)
+      .post("/api/v1/articles")
+      .set({ Authorization: `Bearer ${token}` })
+      .send({ ...article });
+    ({
+      article: { slug }
+    } = articleResult.body);
+  });
+  describe("article fetch and like", () => {
+    describe("Like", () => {
+      it("should allow  user to like an article", done => {
+        chai
+          .request(app)
+          .post(`/api/v1/articles/${slug}/likes`)
+          .set({ Authorization: `Bearer ${token}` })
+          .end((err, response) => {
+            if (err) {
+              return done(err);
+            }
+            expect(response.status).eql(201);
+            expect(response.body).to.an("object");
+            expect(response.body).to.have.property("message");
+            expect(response.body.message).to.be.eql("Successfully liked");
+            done();
+          });
+      });
```

Comment:
Can you please try to keep a single space between the test suits to keep consistency?